locales/sr.json

```diff
@@ -0,0 +1,93 @@
+
+{
+  "ui_use": "Koristi",
+  "ui_give": "Daj",
+  "ui_close": "Zatvori",
+  "ui_drop": "Baci",
+  "ui_removeattachments": "Skini dodatke",
+  "ui_copy": "Kopiraj serijski broj",
+  "ui_durability": "Izdrzljivost",
+  "ui_ammo": "Municija",
+  "ui_serial": "Serijski broj",
+  "ui_components": "Dodaci",
+  "ui_tint": "Tinta",
+  "ui_usefulcontrols": "Korisne kontrole",
+  "ui_rmb": "Otvori kontekst meni",
+  "ui_ctrl_lmb": "Brzo prebaci gomilu predmeta u drugi inventar",
+  "ui_shift_drag": "Podeli broj predmeta na pola",
+  "ui_ctrl_shift_lmb": "Brzo prebaci pola gomile predmeta u drugi inventar",
+  "ui_alt_lmb": "Brzo iskoristi predmet",
+  "ui_ctrl_c": "Dok prelazis misem preko oruzja, kopira njegov serijski broj",
+  "ui_remove_ammo": "Izvadi municija",
```

Comment:
"Izvadi municiju" mislim da ima više smisla.

Reply:
to je i trebalo da bude izvadi municiju :D